test/controllers/RegisteredBusinessControllerSpec.scala

```diff
@@ -81,6 +81,7 @@ class RegisteredBusinessControllerSpec extends PlaySpec with GuiceOneServerPerSu
             document.title() must be("Is this where you want us to send any letters about ATED? - GOV.UK")
             document.getElementById("business-registered-text").text() must be("This section is: ATED registration")
             document.getElementById("registered-business-address-header").text() must be("Is this where you want us to send any letters about ATED?")
+            document.getElementById("backLinkHref").text() must be("Back")
```

Comment:
Also check the back link here is what is in appconfig:
```suggestion
            document.getElementById("backLinkHref").text() must be("Back")
            document.getElementById("backLinkHref").attr("href") must be(backToBusinessCustomerUrl)
```
Requires mocking at the top of the spec...
```
  val backToBusinessCustomerUrl = "someBackToBusinessCustomerUrl"
  when(mockAppConfig.backToBusinessCustomerUrl).thenReturn(backToBusinessCustomerUrl)
```
Also might want to check the back link in the form validation scenarios.

Reply:
Which form validation scenarios where you referring to?

Reply:
Sorry - when it redisplays the page with validation errors. So scenarios: **return to this page if we have an error form validation** and maybe **return to this page if we have an error**

Reply:
I have added these tests